src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/commissioner/thread/ThreadCommissionerException.java

```diff
@@ -0,0 +1,28 @@
+package com.google.chip.chiptool.commissioner.thread;
+
+import androidx.annotation.NonNull;
+
+public class ThreadCommissionerException extends Exception {
+
+  @NonNull private int code;
```

Comment:
should these be final since there are no setters and they are set only in the constructor?